### ISSUE TITLE
Wrong VPN header displayed on the Staging VPN Download page

### ISSUE THREAD

**Issue by LuminitaB** [mozilla/bedrock#13065]
## Description

The Wrong header, with the wrong options, is displayed on the VPN Download page

### Steps to reproduce

1. Access the staging VPN download page
2. Take a look at the displayed Mpzilla VPN header

### Expected result

The correct header is displayed (as seen in [Figma](https://www.figma.com/file/FjeA0eZw4GIxBAlQfz06c1/VPN-Download-page-%5B%2B-Archived-Landing-page%5D?type=design&node-id=2608-7148&t=mNE900A7Hp4jnQKi-0) and on the [Production](https://www.mozilla.org/en-US/products/vpn/download/) page, containing the **Subscribe to Mozilla VPN**  and **Get Help**  options
![image](https://user-images.githubusercontent.com/52028275/235633485-fcb0d2b8-cb3f-4eaf-983a-061d2ada921e.png)

### Actual result

The same header as for the Main page is displayed, with the Mozilla VPN **Learn about VPNs**  and **Download Mozilla VPN**  options
![image](https://user-images.githubusercontent.com/52028275/235633595-a5067d55-c5fb-48d9-b81b-dcab400fa82c.png)

### Environment
Staging

What browser and operating system are you using?
Firefox Release 112.0.2

**Comment by alexgibson** [comment]
/cc @a-kyne @nathan-barrett 

As far as I know this change was intentional, but I'll leave it for you to discuss here.

**Comment by a-kyne** [comment]
Yes! We are updating the subnav in VPN pages. This will 1) create links to the existing "Resource Center" pages so that they are no longer orphaned, and 2) redirect pages that are content duplicates to the RC pages, using a 301 status code. 
Where I have documented my own QA tests? I'm not sure what the appropriate process is here.

**Comment by LuminitaB** [comment]
Thank you for the reply.
So, the Figma and the current Production page are wrong, and the Stage page is correct?

It just seemed weird to me to have a **Download Mozilla VPN** link on the download page, that takes exactly where you already are.

It made more sense to have the **Subscribe to Mozilla VPN** link to take you back to the VPN main page, to get you to give away some money by buying a subscription.

Thank you.

**Comment by alexgibson** [comment]
> So, the Figma and the current Production page are wrong, and the Stage page is correct?

I'll defer to others on which links are most suitable for the subnav on this page, but I can add that the reason the links are different in prod is because we hadn't yet deployed this change at the time the bug was filed.

**Comment by LuminitaB** [comment]
Thank you for looking into this.
Yes, my concern was about the "Download Mozilla VPN" link being redundant, basically acting as a refresh button for the page, and about the "Subscribe to Mozilla VPN" link making more sense in terms of redirecting the user to the subscription page, to actually activate the product they are downloading by getting a subscription.
However, I will change out tests accordingly, just looking for a confirmation of the desired outcome for the subheader menu.
Thank you.

**Comment by alexgibson** [comment]
@a-kyne any thoughts on @LuminitaB's suggestion here?

**Comment by a-kyne** [comment]
@alexgibson @LuminitaB The VPN product pages are being redesigned and will be relaunched with new navigation, among other things; this iteration is only a temporary stopgap, so not really a wontfix. I am confident that the new pages that the Studio designs will incorporate usability best practices.

**Comment by alexgibson** [comment]
I've just seen the proposed new designs for the VPN pages and the sub nav links in the mockups are:

- What's a VPN?
- Pricing
- Features
- Get Help

**Comment by a-kyne** [comment]
Yes, the first round of feedback (e.g. plz add link to the Resource Center) has not been included yet.

**Comment by alexgibson** [comment]
The final links that show in the sub navigation of the newly designed pages can be seen here: https://www.mozilla.org/en-US/products/vpn/

- Learn about VPNs
- Features
- Download Mozilla VPN

It still doesn't contain a "Subscribe", or "Pricing" link, which seems like an odd omission to me - and was also a question by the original bug reporter here. So I'm going to hold off closing this out until we're sure this hasn't slipped through the cracks?

**Comment by noeldlt** [comment]
Hi team,

I'm the new Product Marketing Lead on our PXI portfolio. 

Agree that this is an issue. We're also lacking consistency through the subdomain navs. 

Here's what I'd propose as the short-term solution--we can revisit if/when we have a separate pricing page.

**Learn about VPNs page**
Sub nav: Features | Download Mozilla VPN

**Features page**
Sub nav: Learn about VPNs | Download Mozilla VPN

**Download Mozilla VPN page**
Sub nav: Learn about VPNs | Features
Add "Get Mozilla VPN" CTA in top right corner

Alternatively. we keep all three items across the three subdomain navs and add the "Get Mozilla VPN" CTA in top right corner. Though the link to the same page you're already on (that refreshes the page) seems unnecessary in my opinion.

**Comment by alexgibson** [comment]
@noeldlt i think the attribution issue is not really related to the sub nav links. We can just as easily link to `/products/vpn/#pricing` as we can `products/vpn/pricing/` from the sub nav.

**Comment by alexgibson** [comment]
Related bug I just noticed on the resource center: https://github.com/mozilla/bedrock/issues/13622

**Comment by stephaniehobson** [comment]
I think this was fixed in [#14722](https://github.com/mozilla/bedrock/pull/14722). And I see that there is also styling to indicate the current page though.
